StreamChatClient_v3/WebSocketClient/WebSocketPayload.swift

```diff
@@ -4,30 +4,44 @@
 
 import Foundation
 
-struct WebSocketPayload<ExtraData: ExtraDataTypes>: Encodable {
+struct WebSocketPayload<ExtraData: UserExtraData>: Encodable {
```

Comment:
We need a better name for this, since different websocket payloads will contain different entities

Reply:
This is actually the outgoing payload. It's the initial payload we open the WS connection with. It won't contain different entities. This is it. Any idea for a name?

Reply:
`WebSocketConnectPayload`

Reply:
⬆️ suggestion for a name